.changeset/purple-planets-join.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Adds a new `wrangler pages functions build` command which takes the `functions` folder and compiles it into a single Worker. This was already done in `wrangler pages dev`, so this PR just exposes this build command for use in our build image, or for people who want to do it themselves.
```

Comment:
Can you split this block into two lines, a bit like a commit message, where the first line is the "title" and indicates what changed and the second line is the detail and/or background?

I think the first line reads better if it is imperative.
Also we should talk about changes rather than commits or PRs in changesets, IMO.
Further, since this is a "patch" change rather than a minor, we should avoid making it sound like a feature, if we are going to aim to be semver compliant. (Alternativlely this would need to be marked as a minor feature change.

For example:

```suggestion
Expose `wrangler pages functions build` command, which takes the `functions` folder and compiles it into a single Worker.

This was already done in `wrangler pages dev`, so this change just exposes this build command for use in our build image, or for people who want to do it themselves.
```

Reply:
While we're still in 0.0.X, feature additions are patch changes, no? Ordinarily, this would be a minor bump.

Reply:
👍  right, sorry, I forgot we are in 0.0.x mode.